tests/test_helper/many.bash

```diff
@@ -3,6 +3,22 @@ function pem() {
     echo "$PEM_ROOT/id-$1.pem"
 }
 
+# Print the X-coord of an Ed25519 public key
+function ed25519_x_coord() {
+    openssl pkey -in "$(pem "$1")" -text_pub -noout | grep "    " | awk '{printf("%s ",$0)} END { printf "\n" }' | sed '$s/ *//g' | tr -d ':'
```

Comment:
So this only removes the spaces at the end. If that's your intent you can merge this, but if you want to remove ALL the spaces, you need to remove the `$`.

```suggestion
    openssl pkey -in "$(pem "$1")" -text_pub -noout | grep "    " | awk '{printf("%s ",$0)} END { printf "\n" }' | tr -d ' ' | tr -d ':'
```

Reply:
The `sed` version is working properly.

However, the `tr` version is simpler; I will fix it.